features/security/src/main/feature/feature.xml

```diff
@@ -82,13 +76,6 @@
         <bundle>mvn:ddf.security.encryption/security-encryption-commands/${project.version}</bundle>
     </feature>
 
-    <feature name="clientinfo-filter" version="${project.version}"
```

Comment:
❓ Just want to make sure there isn't any problems with having all of these filters start and not being able to turn them off.

Reply:
People shouldn't be turning these filters off.

Reply:
Sounds good wasn't sure if it mattered or not.